server/portal/apps/tickets/api/views.py

```diff
@@ -70,6 +71,18 @@ def post(self, request):
 
         problem_description += "\n\n" + metadata
 
+        if not request.user.is_authenticated:
+            recap_result = utils.get_recaptcha_verification(request)
+            if recap_result.get('success', False):
+                ticket_id = rt.create_ticket(subject=subject,
+                                             problem_description=problem_description,
+                                             requestor=email,
+                                             cc=cc,
+                                             attachments=attachments)
+                return JsonResponse({'ticket_id': ticket_id})
+            else:
```

Comment:
this could be simplified
```suggestion
            if not recap_result.get('success', False):
```